packages/terra-open-source-scripts/src/terra-cli/prepare-for-release/prepareForReleaseMonoRepoHandler.js

```diff
@@ -0,0 +1,33 @@
+const childProcess = require('child_process');
+const fs = require('fs-extra');
+const path = require('path');
+const { promisify } = require('util');
+const stripAnsi = require('strip-ansi');
+const Logger = require('@cerner/terra-cli/lib/utils/Logger');
+
+const logger = new Logger({ prefix: '[terra-open-source-scripts:version]' });
+
+const exec = promisify(childProcess.exec);
+
+const updateChangelogForPackage = require('./updateChangelogForPackage');
+
+const VERSION_OUTPUT_PATH = path.join(process.cwd(), 'version-output.txt');
+
+module.exports = async () => {
+  try {
+    childProcess.execSync(`script -q ${VERSION_OUTPUT_PATH} lerna version --no-git-tag-version`, { stdio: 'inherit' });
```

Comment:
Feels like lerna should be a dependency if we are going to use it here

Reply:
If you're a mono repo you already have lerna, if your not this won't get called. I'm ok with this being loosely coupled.

Reply:
Do we have to write this file to disk? does the script output not get returned by the execSync call?

Reply:
It does not return here. I will document this, but this particular functionality was somewhat difficult to implement.  

The only way I could properly get interactive behavior with this command was to use stdio 'inherit'. Once you do that it no longer returns the output from the command. Even doing something like `> <file>` doesn't really work the way that it needs to. `script` seemed to be the only thing that got me what I needed.